app/src/main/java/com/kickstarter/viewmodels/BackingAddOnsFragmentViewModel.kt

```diff
@@ -131,25 +145,23 @@ class BackingAddOnsFragmentViewModel {
                 .map { it.reward() }
 
             val backing = projectData
-                .map { getBackingFromProjectData(it) }
-                .filter { ObjectUtils.isNotNull(it) }
-                .map { requireNotNull(it) }
+                .filter { getBackingFromProjectData(it) != null }
+                .map { requireNotNull(getBackingFromProjectData(it)) }
```

Comment:
with the migration any RXchain that could emmit a null needs to be solved or the `java.lang.NullPointerException: The mapper function returned a null value` will the thrown, the aking object when it's a new pledge it's null so we filter first, and emmit in case there is backing information. With RXJava1 there were no issues at all with emmiting null objects